Sources/MapboxDirections/Intersection.swift

```diff
@@ -292,10 +317,23 @@ extension Intersection: Codable {
         if let lanes = try container.decodeIfPresent([Lane].self, forKey: .lanes) {
             approachLanes = lanes.map { $0.indications }
             usableApproachLanes = lanes.indices { $0.isValid }
+            preferredApproachLanes = lanes.indices { ($0.isActive ?? false) }
+            
+            let temp = LaneIndication(rawValue: 0)
```

Comment:
This constant seems to only be used as a temporary base case to reduce into. We can inline it; it’s no problem to have a slightly longer reduce statement as a result.